mysql-test/suite/galera/t/MDEV-30804.test

```diff
@@ -0,0 +1,21 @@
+#
+# Test that transaction requiring two-phase commit and involving
+# storage engines not supporting it rolls back with a message.
+#
+
+--source include/galera_cluster.inc
+--source include/have_innodb.inc
+--source include/have_aria.inc
+
+CREATE TABLE t (a INT) ENGINE=Aria;
+CREATE TABLE t1 (a INT);
+
```

Comment:
InnoDB is currently the defalut storage engine, but for readability, it is better to declare table t1 to have ENGINE=InnoDB